packages/ui-predicate-react/src/UIPredicateCoreReact.js

```diff
@@ -0,0 +1,57 @@
+/* eslint no-unused-vars: "off"*/
+
+/**
+ * React adapter for ui-predicate-core
+ * @namespace react
+ * @since 0.2.0
+ */
+
+import React from 'react';
+import { PredicateCore, UITypes } from 'ui-predicate-core';
+import assign from 'lodash/assign';
+import DEFAULT_COMPONENTS from './default-components';
+
+const defaults = {
+  options: {
+    /**
+     * UIPredicate React Adapter own default argument component
+     * @param {Object} [columns=PredicateCore.defaults.columns] columns
+     * @param {Object} [options=PredicateCore.defaults.options] options
+     * @param {Object} [ui=PredicateCore.defaults.ui] ui
+     * @return {Vue.component} the default Vue Component to use as argument specifier
+     * @see core.defaults.getDefaultArgumentComponent
+     * @memberof vue.defaults
```

Comment:
Does it should belongs to react not vue ?

Reply:
Ooh yeah. That's was an incorrect copy-paste. I'll fix it.